dune_api_scripts/store_query_result_all_distinct_app_data.py

```diff
@@ -1,38 +1,34 @@
 """
-Stores the result of querying all distinct app data in a file `distinct_app_data.json`
+Queries and stores all distinct app data in a file `distinct_app_data.json`
 """
 import json
 import os
+import time
 from pathlib import Path
-
-from .utils import dune_from_environment, parse_dune_iso_format_to_timestamp
+from duneapi.api import DuneAPI
+from duneapi.types import DuneQuery, Network
+from duneapi.util import open_query
 
 if __name__ == "__main__":
-    entire_history_path = Path(os.environ['DUNE_DATA_FOLDER'] + "/app_data/")
+    entire_history_path = Path(os.environ["DUNE_DATA_FOLDER"] + "/app_data/")
     os.makedirs(entire_history_path, exist_ok=True)
 
     # initialize the environment
-    dune = dune_from_environment()
+    dune = DuneAPI.new_from_environment()
 
     # fetch query result id using query id
-    query_id = int(os.getenv('QUERY_ID_ALL_APP_DATA', "142824"))
-    result_id = dune.query_result_id(query_id)
+    query_id = int(os.getenv("QUERY_ID_ALL_APP_DATA", "142824"))
+    query = open_query("./dune_api_scripts/queries/all_app_data.sql")
+    time_of_request = int(time.time())
+    dune_query = DuneQuery("", "", query, Network.MAINNET, [], query_id)
```

Comment:
For debugging it was always nice to have an url that one can visit. But yeah, you are right that it is no strictly necessary. 
Since the one script takes quite a while and the other one is super quick, sometimes one would have to run them in parallel and then its nice to have two different query_ids, such that the queries do not overwrite themselves.